worker-store-fs/src/main/java/com/hpe/caf/worker/datastore/fs/FileSystemDataStore.java

```diff
@@ -65,11 +65,12 @@ public FileSystemDataStore(final FileSystemDataStoreConfiguration config)
         throws DataStoreException
     {
         dataStorePath = FileSystems.getDefault().getPath(config.getDataDir());
-        throwExceptionIfPathDoesNotExist(dataStorePath);
-        try {
-            Files.createDirectory(dataStorePath);
-        } catch (IOException e) {
-            throw new DataStoreException("Cannot create data store directory", e);
+        if (!Files.exists(dataStorePath)) {
```

Comment:
@dermot-hardy Do we need to replace this as well? Here non-existence is expected whereas in places where we are replacing Files.exists, existence of path is expected. If we still want to replace this, then we may need to gobble up the exception and just log it. 

Let me know what you think.